src/components/Avatar/Avatar.jsx

```diff
@@ -190,7 +190,7 @@ const NonAvatarContainer = styled(Flex)(
   ({ variant }) => css`
     overflow: hidden;
     cursor: pointer;
-    border-radius: ${variant ? 2 : '50%'};
+    border-radius: ${variant ? 4 : '50%'};
```

Comment:
Aqui não funciona passar so o valor 4, teria que adicionar o "px" para funcionar, mas é melhor usar o space direto do theme, então na linha 190 utilizar depois do variant `theme: { space }` e utiliza ele para chamar o valor 4 direto do theme.
```suggestion
    border-radius: ${variant ?  `${space[2]}px` : '50%'};
```